chunk/chunk.go

```diff
@@ -278,7 +278,8 @@ type Validator interface {
 // with validators check.
 type ValidatorStore struct {
 	Store
-	validators []Validator
+	pssCallback func(Chunk) error
```

Comment:
i think this callback is not pss related from the store point of view, better call it
`deliverHook` or `deliverCallback` maybe .
i dont think it needs to return anything

Reply:
agreed, poorly chosen name.

fixed